lib/exrabbit/util.ex

```diff
@@ -87,6 +87,9 @@ defmodule Exrabbit.Util do
   end
 
   defp get_formatter_module(atom) do
-    Module.concat([Exrabbit.Formatter, Atom.to_string(atom) |> String.upcase()])
```

Comment:
I just don't like the way how that wrapper would be implemented. I mean without this change users should write Exrabbit.Formatter.<FORMAT> wrappers which would produce Elixir.Exrabbit.Formatter.<FORMAT>.beam module, which looks like a module from Exrabbit library, but it's not.
Also, not having a wrapper around the formatted is better than having it.